src/python_freeipa/client.py

```diff
@@ -110,6 +110,27 @@ def _request(self, method, args=None, params=None):
         else:
             return result['result']
 
+    def change_password(self, username, new_password, old_password):
+        """
+        Set the password of a user. (Does not expire)
+
+        :param login: User login (username)
```

Comment:
@Epictek It looks like username is not used at all in this method, so it doesn't make sense to require this parameter. As I understand this request modifies password for the current user instead of user identified by username, so this is really misleading. Therefore it makes sense to rename this method and update it's parameters so that it's name would match its behavior.

Reply:
@Epictek poilte ping, are you still working on this merge request?

Reply:
Yes, sorry forgot about this. the params  require the username of the current logged in user (at least this is what the FreeIPA web interface uses).
`data = {'new_password': new_password, 'old_password': old_password, 'user':username}`
But I can test what happens if the username is not given in the request.

Also I agree on the function renaming.